src/hooks/tests/use-site-sync-management.test.ts

```diff
@@ -0,0 +1,142 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { useAuth } from '../use-auth';
+import { useFetchWpComSites } from '../use-fetch-wpcom-sites';
+import { useSiteDetails } from '../use-site-details';
+import { useSiteSyncManagement } from '../use-site-sync-management';
+
+jest.mock( '../use-auth' );
+jest.mock( '../use-site-details' );
+jest.mock( '../use-fetch-wpcom-sites' );
+
+export const mockConnectedWpcomSites = [
+	{
+		id: 6,
+		localSiteId: '788a7e0c-62d2-427e-8b1a-e6d5ac84b61c',
+		name: 'My simple business site',
+		url: 'https://developer.wordpress.com/studio/',
+		isStaging: false,
+		stagingSiteIds: [ 7 ],
+		syncSupport: 'syncable',
+	},
+	{
+		id: 7,
+		localSiteId: '788a7e0c-62d2-427e-8b1a-e6d5ac84b61c',
+		name: 'Staging: My simple business site',
+		url: 'https://developer-staging.wordpress.com/studio/',
+		isStaging: true,
+		stagingSiteIds: [],
+		syncSupport: 'syncable',
+	},
+];
+
+export const ConnectedSitesStore = {
```

Comment:
What is the reason to export it too?
And how it work, if we don't use it inside of this module? Looks totally unclear and magically 😅

Reply:
Yeah, I believe that this is not really necessary. I removed it in https://github.com/Automattic/studio/pull/641/commits/c6732abaff4e1e938177680fdde1862fef35e7f7

A bit of background why this happened is that I initially was planning to use a more complex data structure for these tests but it turned out to be not necessary.